### ISSUE TITLE
[Linux] Crash when opening control panel

### ISSUE THREAD

**Issue by mstarongithub** [TASEmulators/desmume#731]
In the flathub version, when opening `Config -> Edit Controls`, it crashes.
Output log of running in console:
```
Microphone successfully inited.
DeSmuME 0.9.14 git#0 x64-JIT SSE2
CPU mode: Interpreter
Using 12 threads for video filter.
SoftRasterizer: Running using 12 additional threads. (Multithreading enabled.)
*** buffer overflow detected ***: terminated
```

**Comment by ofdx** [comment]
Also seeing this. Crashes immediately upon clicking Edit Controls, same CLI output. Flathub release, running on debian trixie.

I built from source and could not reproduce the crash. I think that buffer overflow detected message can be generated by printf, so I tried to modify the call to g_strdup_printf to induce a crash (NULL arguments, large values and negative values into gdk_keyval_name to produce bad strings), but I couldn't break it. Maybe a library version mismatch in the flatpak?

**Comment by macschaff** [comment]
Having the same issue on Steam OS on the Steam Deck.

**Comment by rofl0r** [comment]
build from source with debug info, then provide a gdb backtrace from the crash.

**Comment by zShxck720** [comment]
I'm having the same problem on Debian Bookworm

**Comment by rofl0r** [comment]
with flathub, too ? then maybe you should raise the issue with the flathub package maintainer.

**Comment by zShxck720** [comment]
> then maybe you should raise the issue with the flathub package maintainer

Yes with the flatpak. The problem is that on Flathub there is written to report an issue on this website. I don't know how to contact the flatpak manteiner

**Comment by rofl0r** [comment]
try the commands mentioned by the buildbot here: https://github.com/flathub/org.desmume.DeSmuME/pull/14 - that would also be the correct place to report issues with the flatpak.

**Comment by intra0** [comment]
this issue should be closed and merged with #843 


also the other currently open flatpak issue #513 has been fixed and should also be closed
